artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -3164,6 +3174,7 @@ private void checkDepage() {
          return;
       }
       if (pageIterator != null && pageSubscription.isPaging()) {
+         logger.debug("CheckDepage on queue name {}, id={}", this.name, this.id);
```

Comment:
The "this." qualifiers seem unnecessary.

Reply:
@gemmellr I actually like the this quite a lot.. makes it explicitly and where it's coming from when you read it.

I guess it will be personal taste here?